portfolio/src/main/java/com/google/sps/servlets/ChooseUsernameServlet.java

```diff
@@ -53,7 +53,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     TransactionOptions options = TransactionOptions.Builder.withXG(true);
     Transaction transaction = datastore.beginTransaction(options);
     try {
-      Entity usr = new Entity("Username", username);
       Entity check = null;
       try {
         Key usrkey = KeyFactory.createKey("Username", username);
```

Comment:
(this is meant for one of the lower lines but I can't click on there)

Is there a way to check if the entity was returned, rather than relying on catching an exception? It's extremely rare that it's desirable to swallow an exception because it's what we want, because it makes things harder to read.

Reply:
+1